orbit-2-test/src/test/java/com/babylon/orbit2/OrbitTestingTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Babylon Partners Limited
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.babylon.orbit2
+
+import com.nhaarman.mockitokotlin2.mock
+import org.junit.jupiter.api.Test
+
+class OrbitTestingTest {
+    @Test
+    fun `basic test`() {
+
+        val mockDependency = mock<BogusDependency>()
+        val testSubject = MyClass(mockDependency)
+
+        testSubject.something(true)
+
+        val given = testSubject.given(State())
```

Comment:
Is it worth removing the intermediary vals here?

testSubject
    .given...
    .whenever..